src/sugar3/bundle/__init__.py

```diff
@@ -46,25 +47,31 @@
 
 Optional metadata keys are;
 
-* `summary` - a short summary of the activity that may be displayed in the List or Home Views,
+* `summary` - a short summary of the activity that may be displayed in
+  the List or Home Views,
 
 * `mime_types` - list of MIME types supported by the activity,
   separated by semicolons.  Your `read_file` method must be able to read
-  files of these MIME types.  Used to offer your activity when opening a
-  downloaded file or a journal object.
+  files of these MIME types.  Used to offer your activity in a list of activities
+  when opening a downloaded file from the web or a certain journal object.
 
 * `url` - link to the home page for the activity,
 
 * `repository` - link to repository for activity code, for use by git clone,
 
 * `single_instance` - if yes, only a single instance of an activity
-  should be started, and if another instance is requested the existing
-  instance shown,
+  should be started at any one time, and if another instance is requested
+  the existing instance shown,
 
-* `max_participants` - maximum number of participants for sharing an activity,
+* `max_participants` - maximum number of participants that can join a shared activity,
 
 * `tags` - more context into which to place an activity,
 
+* `category`/`categories` - define/choose a category/categories that fits your
+activity suitably. You can choose from the list available at http://activities.sugarlabs.org/en-US/sugar/,
+
+* `website` - link to activity-documentation as present in help-activity,
```

Comment:
We already have `url`, so you are adding an alias for `url`, right?

Perhaps additionally describe `url` as "or link to the activity on https://help.sugarlabs.org/ "

Reply:
Yes, in the case if the activity already has an `url` defined, I didn't want to overwrite anybody's else work. Additionally described `url` as well.